hist/histv7/inc/ROOT/RAxis.hxx

```diff
@@ -759,6 +790,46 @@ public:
          vec.at(kv.second) = kv.first;
       return vec;
    }
+
+   /// Result of an RAxisLabels label comparison
+   enum LabelComparisonFlags {
```

Comment:
`LabelComparisonFlags` isn't really about comparing **a** label. This enum is already in `RAxisLabels`, so maybe just `RAxisLabels::ComparisonFlags`?

Reply:
What if we latter decide to add a more general fuzzy comparison operator to RAxisBase, which needs its own `ComparisonFlags` enum?

Reply:
Maybe `LabelsCmpFlags`? I'd like to make sure it's clear that we are not doing substring comparisons here: we do not compare individual *label*s but *sets* of labels.

Reply:
Switched to this naming convention in the latest commit.